docs/17-porting-tips.md

```diff
@@ -9,7 +9,7 @@ you might have added to your `rc.lua`.
 This avoids most of the possible errors due to missing important changes.
 
 If you still wish to ignore this advice, first read the
-<a href="89-NEWS.md.html#v4">NEWS</a> section about the breaking changes. This
+[NEWS](89-NEWS.md) section about the breaking changes. This
```

Comment:
Oh, sorry, my bad. I tried to open the link out of wiki and decided it's broken.